runtime.go

```diff
@@ -1,20 +1,19 @@
 package runtime
 
 import (
-	"github.com/ahab94/ApnaSchool/db"
-	"github.com/ahab94/ApnaSchool/db/mongo"
+	"github.com/ahab94/ApnaSchool/service"
 )
 
 // Runtime initializes values for entry point to our application
 type Runtime struct {
-	Db db.DataStore
+	svc *service.Service
 }
 
-// NewRuntime creates a connection to our database
+// NewRuntime creates a new database service
 func NewRuntime() (*Runtime, error) {
-	store, err := mongo.NewClient(db.Option{})
+	s, err := service.NewService()
 	if err != nil {
 		return nil, err
 	}
-	return &Runtime{store}, err
+	return &Runtime{s}, err
```

Comment:
It will be better to explicitly instantiate variable. like 	`return &Runtime{svc: svc}, err`